scripts/dev/license-check.py

```diff
@@ -101,7 +101,8 @@
 # Check files
 python_file_license_success = True
 patterns = [r'.*third_party.*', r'^\.(\\|/)build.*',
-            r'^\.(\\|/)bin.*', r'.*readthedocs.*']
+            r'^\.(\\|/)bin.*', r'.*readthedocs.*',
+            r'.*venv.*', r'.*cmake-build-.*']
```

Comment:
The license check script was throwing warnings about two directories:
- `repo_root/venv`: an exceedingly common place for a Python virtual environment
- `repo_root/cmake-build-debug`: where CLion creates a build folder -- also can be `cmake-build-release` and others, so just matching on the base pattern.